ovos_backend_client/pairing.py

```diff
@@ -72,6 +79,10 @@ def check_remote_pairing(ignore_errors, url=None, version="v1", identity_file=No
     Returns:
         True if pairing checks out, otherwise False.
     """
+    backend_type = backend_type or Configuration()["server"].get("backend_type") or BackendType.OFFLINE
```

Comment:
Should this be cast to BackendType()? I think inbound strings will fail the following check, without the cast

Reply:
Can that arg also be added to the docstring with a type annotation to catch type errors in outside calls?

Reply:
BackendType should be a valid string, subclasses both from str and enum, i recall the ordering of subclassing was important, perhaps something changed or there is a bug in BackendType class definition